packages/compiler-sfc/__tests__/compileScript.spec.ts

```diff
@@ -565,6 +566,8 @@ defineExpose({ foo: 123 })
       expect(content).toMatch(`unref(bar)`)
       // should unref other imports
       expect(content).toMatch(`unref(other)`)
+      // #6483 should add an extra set of parentheses after new
+      expect(content).toMatch(`new (_unref(TestClass))()`)
```

Comment:
I think it should be generated as `_unref(new TestClass())` because the constructor of TestClass may return a ref value.

Reply:
As you think it should be`_unref(new (_unref(TestClass))())`, because `TestClass` may be a ref

Reply:
@edison1105 I resolved the issue of #6843 and the issue you raised